src/components/Records/Record/GeneralInfo/Maintainers.vue

```diff
@@ -22,7 +27,13 @@
           v-if="!canClaim && !user().isLoggedIn"
           class="ma-0 mr-1"
         >
-          and claim it now!
+          and
+        </p>
+        <p
+          v-if="!canClaim"
+          class="ma-0 mr-1"
+        >
+          claim it now!
```

Comment:
Probably better to remove this phrase altogether if this is the way you'd rather do it. So, instead of:

`This record is in need of a maintainer. claim it now!` (note dodgy capitalisation)

...it would be:

`This record is in need of a maintainer.`